packages/cli/src/commands/database/seed/oidc-config.ts

```diff
@@ -116,7 +120,11 @@ export const oidcConfigReaders: {
   },
   [LogtoOidcConfigKey.CookieKeys]: async () => {
     const envKey = 'OIDC_COOKIE_KEYS';
-    const keys = getEnvAsStringArray(envKey);
+    const keys = getEnvAsStringArray(envKey).map((key) => ({
+      id: generateStandardId(),
+      value: key,
+      createdAt: Math.floor(Date.now() / 1000),
```

Comment:
nit: can make this a util function with a simple input parameter `key`

Reply:
updated